friends/src/App.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import './App.css';
+import FriendList from './Component/FriendList';
+import axios from 'axios';
+
+class App extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      friends: [],
+      id: '',
+      name: '',
+      age: '',
+      email: '',
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get(`http://localhost:5000/friends`)
+      .then(response => {
+        console.log('This is the axios');
+        console.log(response.data);
```

Comment:
don't forget to remove the `console.log`s you don't need any more, once it works 🕵️